src/main/java/pl/nyczjablonska/recursion/Factorial.java

```diff
@@ -2,6 +2,9 @@
 
 public class Factorial {
     public static long calculateNumberFactorial(int number){
+        if (number < 0) {
```

Comment:
good condition! let's think how can we simplify this condition together with the later one now:)